src/style.css

```diff
@@ -1 +1,59 @@
-/* add styles here */
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+  margin: 0;
+  padding: 0;
+}
+
+.stars__star {
+  background-image: url(images/star.svg);
+  width: 16px;
+  height: 16px;
+  margin: 0 2px;
+  padding: 0;
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars--0 .stars__star {
+  margin-top: 0;
+}
```

Comment:
The `margin-top: 0;` rule in `.stars--0 .stars__star` seems unnecessary, because you've already set `margin: 0;` in the `.stars__star` rule. Consider removing it to keep your CSS clean and efficient.